site/docs/ch/benchmark.md

```diff
@@ -0,0 +1,115 @@
+#Lakehouse 性能对比测试报告
+##测试环境
+
+###机器环境
+
+| 编号 | 操作系统 | cpu线程数 | 内存 | 磁盘类型 |
+| ---- | -------- | --------- | ---- | -------- |
+| 1    | CentOS 7 | 40        | 256  | SAS      |
+| 2    | CentOS 7 | 40        | 256  | SAS      |
+| 3    | CentOS 7 | 40        | 256  | SAS      |
+| 4    | CentOS 7 | 40        | 256  | SAS      |
+| 5    | CentOS 7 | 40        | 256  | SAS      |
+| 6    | CentOS 7 | 40        | 256  | SAS      |
+
+###软件环境
+
+Trino版本：380(用于iceberg和arctic的测试)
+
+Presto: 274(用于Hudi的测试，Trino不支持Hudi的MOR实时查询)
+
+Iceberg版本：0.13  （由于开源的iceberg在trino查询中的性能非常低，所以后面的测试均采用的是内部优化过后的版本，相关优化也提交社区见: https://github.com/apache/iceberg/issues/5245, https://github.com/apache/iceberg/pull/5264）
+
+Arctic版本：0.3
+
+1，2，3机器部署hadoop集群包括hive，yarn，hdfs，最终iceberg和arctic的数据是落在这三台机器上并且负责flink写入任务，mysql服务。
+
+4，5，6机器部署查询Trino查询引擎，部署了3个节点每个节点设置64G内存。
+
+##测试方案
+
+###CHbenchmark
+
+本次测试规范基于CHbenchmark，CHbenchmark是一个集成了TPCC和TPCH的混合测试标准，测试负载整体上可分成两类：
+
+- 基于TPC-C的5个OLTP型负载：NewOrder, Payment, OrderStatus, Delivery和StockLevel
+
+- 基于TCP-H改写的22个OLAP型负载，其中由于Q15查询和视图相关，故此次测试舍弃了Q15
+
+![OLTP AND OLAP](images/OLTP-OLAP.png)
+
+###基于TPC-C的数据构造
+
+基于TPC-C标准，本次测试在 MySQL中构造了原始数据，数据集总共包括12张表，其中TPC-C和TPC-H表的关系如下图所示：
+
+![TPCC AND TPCH](images/TPCC-TPCH.png)
```

Comment:
Images should be moved into `docs/ch/images` folder.
And I suggest to make a new folder under images to place the benchmark images.